### DISCUSSION TITLE
HTML/CSS autocompletion isn't working for me

### DISCUSSION THREAD

**Discussion by lemonadeforlife** [neovim/neovim#27879] [Q&A]
I'm using [nvim-cmp](https://github.com/hrsh7th/nvim-cmp) and [luasnip](https://github.com/L3MON4D3/LuaSnip).

My lsp configuration is [here](https://github.com/lemonadeforlife/nvim_config/blob/dev/lua/nahian/plugins/lsp/lspconfig.lua)

**Comment by clason** [comment]
Please report your issue there, or ask on Matrix. This is not a personal support forum.
